src/SFA.DAS.EAS.Employer_Account.Database/StoredProcedures/EvaluateEmployerLegalEntityAgreementStatus.sql

```diff
@@ -0,0 +1,50 @@
+﻿CREATE PROCEDURE [employer_account].[EvaluateEmployerLegalEntityAgreementStatus]
```

Comment:
This sproc is not unit testable but I understand why you've done it this way instead of a couple of round trips to the DB.

Reply:
Okay